app/controllers/publish/recruitment_cycles_controller.rb

```diff
@@ -0,0 +1,14 @@
+module Publish
+  class RecruitmentCyclesController < PublishController
+    def show
+      authorize provider, :show?
+
+      @recruitment_cycle = RecruitmentCycle.find_by(year: params[:year])
```

Comment:
Can we re-use these two assignments from the parent controller? https://github.com/DFE-Digital/teacher-training-api/blob/master/app/controllers/publish/publish_controller.rb#L11

Reply:
The params passed are #<ActionController::Parameters {"controller"=>"publish/recruitment_cycles", "action"=>"show", "provider_code"=>"D87", "year"=>"2022"} permitted: false>` so it needs the `year` not `recruitment_cycle_year`

Reply:
I think we should be able to do something like:

`params[:recruitment_cycle_year] || params[:year] || Settings.current_recruitment_cycle_year` in the parent controller but that might start to look a little obfuscated